lang/en.json

```diff
@@ -102,6 +101,8 @@
     "SETTINGS.RangeAutoCalculationHint" :  "Prefill the roll dialog with a modifier based on the distance between tokens.",
     "SETTINGS.PlayerExperienceEditing" : "Player Experience Editing",
     "SETTINGS.PlayerExperienceEditingHint" :  "Allows players to manually edit their experience point values.",
+    "SETTINGS.UnofficialGrimoire" : "Unofficial Grimoire Magic",
+    "SETTINGS.UnofficialGrimoireHint" : "This is my variation on The ruleset from the Unofficial Grimoire. It includes Catastrophic Manifestations of Chaos, similar to MooMan's house rules, and a change in the use of ingredients. Ingredients react differently depending on how you channel your magic through them. Before casting or channeling, you have to decide whether you want to use the ingredient for Power or Control. Ingredients are consumed even if a spell fails. <br/> <b> Power Component </b>: when used in a Channeling Test, the Power Component doubles the number of PS gained on passing the test (see Power Transfer). During a Cast Test, the Power Component allows you to flip the dice and alter the result of the test. However, if the value on the Unity die after reversing the Spellcasting Test exceeds your Willpower bonus, the energy bursts on you and releases a Minor Chaos Manifestation. <br/> <b> Control Component </b>: When used in a Channeling or Spell Casting Test, the control component acts as a fuse for out of control magic. Reduce the weight of one of the manifestations by one level: i.e., Catastrophic to Major, Major to Minor, or Minor to none. <br/> <b> Power Transfer </b>: during Channeling, it is assumed that you are unloading excess PS beyond what is required harmlessly reducing the spell's health to 0. After performing the test, you may instead save the excess magical energy you have accumulated and apply it to the Transfer of Power when Casting a Spell. (Example: Bianca casts a spell with CN 3 and gains 7 SL; after casting a spell, she decides to collect an additional 4 SL for the effects of a Critical Cast.) But this extra power comes at a cost. If the Spell Casting Test fails, you will lose all energy and trigger a Minor Manifestation. Furthermore, any double or any roll ending in 0 for a Language (Magic) skill is treated as Bad Luck (i.e. 00, 99, 90, 88 etc), and any Bad Luck is treated as a Major Manifestation instead of a Minor. Risky Casting: If you sacrifice Movement and all free actions, you can use your turn to quickly Channell and Cast a Spell in the same action. Because you are in a hurry, increase the weight of each Manifestation you evoke by one level (ie, From Minor to Major or Major to Catastrophic).",
     "Configure": "Configure",
```

Comment:
Can this be a link to the information (such as the wiki or the pdf itself) instead of a text dump in the settings?

Reply:
Sure, didn't think about it. I will prepare something for wiki and find the link to pdf itself

Reply:
I've made a copy of your wiki and added suggested changes here: https://github.com/silentmark/wfrp4e-wiki/blob/master/Homebrew-Rules.md#magic-insiperd-by-unofficial-grimoire

I think there is no way to create a PR to github wiki

Reply:
The wiki should be publicly editable, is it not?

Reply:
If it is, I couldn't find a way to modify it directly

Reply:
Anyway, let me know if there could be any improvements or changes on my side